wiki/core/markdown/mdx/responsivetable.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import, unicode_literals
+
+import markdown
+from markdown.util import etree
+from markdown.treeprocessors import Treeprocessor
+
+
+class ResponsiveTableExtension(markdown.Extension):
+    """Wraps all tables with Bootstrap's table-responsive class"""
+
+    def extendMarkdown(self, md, md_globals):
+        md.treeprocessors.add('responsivetable', ResponsiveTableTree(md), "_end")
+
+
+class ResponsiveTableTree(Treeprocessor):
+    def run(self, root):
+        for table_wrapper in list(root.getiterator('table')):
+            table_new = self.create_table_element()
+            self.convert_to_wrapper(table_wrapper)
+            self.move_children(table_wrapper, table_new)
```

Comment:
I can't think of a case where it would. 

The logic is like this: 

- Create new table element
- Move children (thead, tbody) from old table to new table
- Append new table to old table
- Rename old table tag to div and add Bootstrap class

It's done this way to preserve parent and children of the table.
There might be better ways to add elements in the middle of an ElementTree, but I couldn't find any.